### PR TITLE
PR: Displayed the 20 NFTs from the API

### PR DESCRIPTION
**Pull request by ZinedineDumas**
- call the API
- show the first 20 listing.

### CODE REVIEW

**Comment by skygentleman** [comment]
After `-` there should be a space.

**Comment by ZinedineDumas** [comment]
> Owner



> Ow

**Comment by ZinedineDumas** [comment]
I resolved that quiz.

**Comment by ZinedineDumas** [comment]
I resolved that problem

File: src/pages/NFTS/NFTs.component.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  const [listing, setListing] = useState([])
+
+  const loadListing = async () => {
+    const response = await fetch(
+      `https://api-mainnet.magiceden.io/idxv2/getListedNftsByCollectionSymbol?collectionSymbol=okay_bears&limit=20&offset=0`
+    )
+    const data = await response.json()
+    setListing([...listing, ...data.results])
+
+    console.log(response)
```
Comment (skygentleman) [inline src/pages/NFTS/NFTs.component.js:18]: remove `console`

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,25 @@
-import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
```
Comment (skygentleman) [inline src/App.js:1]: update the prettier setting from single quote to double quote. your setting is different from @skygentleman's

Comment (ZinedineDumas) [reply src/App.js:1]: I removed the console statement in this code.

File: src/pages/NFTS/NFTs.component.js (hunk 1 above)
Comment (ZinedineDumas) [reply src/pages/NFTS/NFTs.component.js]: I removed the console statement in this code.

File: src/App.js (hunk 2 above)
Comment (ZinedineDumas) [reply src/App.js]: Changed the single quotes to the two quotes

**Comment by ZinedineDumas** [comment]
I corrected the mistakes and removed the unnecessary part of this code

File: src/App.js (hunk 2 above)
Comment (ZinedineDumas) [reply src/App.js]: Changed the single quotes to the two quotes

Comment (ZinedineDumas) [reply src/App.js]: Inputted the colon in this code.

**Pull request merged**